views/partials/head.ejs

```diff
@@ -0,0 +1,19 @@
+<meta charset="UTF-8">
```

Comment:
Nice work on the partials. A little un-intuitive though. You  **need** to make advance topics ie partials, more intuitive for others.